providers/attio/parse.go

```diff
@@ -0,0 +1,43 @@
+package attio
+
+import (
+	"strconv"
+
+	"github.com/amp-labs/connectors/common"
+	"github.com/amp-labs/connectors/common/urlbuilder"
+	"github.com/spyzhov/ajson"
+)
+
+func makeNextRecordsURL(reqLink *urlbuilder.URL) common.NextPageFunc {
+	return func(node *ajson.Node) (string, error) {
+		url, err := reqLink.ToURL()
+		if err != nil {
+			return "", err
+		}
+
+		previousStart := 0
+
+		// Extract the data key value from the response.
+		value := node.MustObject()["data"].MustArray()
+
+		if (url.Query().Has("limit") || url.Query().Has("offset")) && len(value) != 0 {
```

Comment:
I think it is worth changing urlbuilder.URL to address this issue instead of converting to url.URL.
urlbuilder holds query parameters already:
```
type URL struct {
	delegate           *url.URL
	queryParams        url.Values
	encodingExceptions map[string]string
}
```
Just add `HasQueryParam(name string)` method and delegate the call `u.queryParams.Has(name)`.